daffodil-core/src/main/scala/org/apache/daffodil/dsom/ChoiceGroup.scala

```diff
@@ -94,8 +94,7 @@ trait ChoiceDefMixin
   }
 }
 
-abstract class ChoiceTermBase(
-  final override val xml: Node,
+abstract class ChoiceTermBase(final override val xml: Node,
```

Comment:
This is a whitespace change. Apologies in advance for these. Even with the diff settings "Hide whitespace changes" these show up.

Reply:
From sclariform on github:

alignParameters

Default: false

Align class/function parameters (modifiers and name, type, and defaults) in three columns.

For example, if false, then:

class Person(name: String,
  age: Int = 24,
  birthdate: Date,
  astrologicalSign: String = "libra",
  shoeSize: Int,
  favoriteColor: java.awt.Color
)

That appears to be the default behavior, and what my eclipse is doing/enforcing. I have to check a box to get it to put the first parameter on a separate line.

Reply:
Wierd that that's the default, the Scala Style Guidelines are very clear about wrapping and indentation:

https://docs.scala-lang.org/style/indentation.html

Reply:
I didn't find anything in the scala style site about method argument declarations. 

They're clear about Constructor arg declarations, and I don't see why method argument declarations wouldn't follow the same rule. But the style site doesn't say, and Scalariform doesn't do it that way by default. 

Personally, I think args should all go on one line unless the line is too long in which case they should go like Constructor args.  I am not sure any of the automated tools would do this.

Reply:
New Info: so the scalariform parameter of interest is actually not the one mentioned above. It is firstParameterOnNewline which defaults to "Force" which puts a new line before the first parameter every time, even if all the args would just fit on one line. There is also Prevent, which is first arg stays on the line, and Preserve, which doesn't futz with them. However, eclipse just lets you check a box or not, so you have Force, or Preserve as the settings I believe. 

Turns out in eclipse, if you choose "reset to defaults" on the scala formatter, it checks the box for this. I was previously under the impression that default settings had no boxes checked, but that is not the case. 

So I will reset these to the default, and re-save the files I have been editing to reapply the first parameter on a new line.

Reply:
Yet more annoying new info.

Eclipse default do not match the scalariform documentation for defaults.

In particular, eclipse defaults to aligining the arrows of case statements. But this is false by default in scalariform. 
One must choose eclipse defaults, then explicitly unset this box.

Reply:
One more time. There are TWO boxes in eclipse that are checked by default and are non-standard (do not match what scalariform claims is its default behavior).

One is to not align case arrows
The other is to not align parameter types.